Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -716,6 +724,14 @@ static Color activeButtonTextColor()
             return semanticColorFromNSColor(alternateColors[1]);
         }
 
+#if HAVE(NSCOLOR_FILL_COLOR_HIERARCHY)
```

Comment:
This should have been `!HAVE`, but I think we can just remove now, based on the discussion below.